relay/sources/rudder-server-relay-postinst

```diff
@@ -49,8 +49,10 @@ chgrp rudder-policy-reader /var/rudder/share
 chmod g+s-w /var/rudder/share
 
 # change some directory to rudder owner
+mkdir -p /var/rudder/configuration-repository/shared-files
```

Comment:
This breaks on relay without root server. Should be done in rudder-webapp

Reply:
ok i was scared because of https://issues.rudder.io/issues/17884 but the mkdir should be ok.

This file is created when the relay synch a shared-file, now it will be created by packaging